pettingzoo/utils/conversions.py

```diff
@@ -32,6 +32,22 @@ def from_parallel(par_env):
         return ordered_env
 
 
+def to_parallel(aec_env):
```

Comment:
Do we really want legacy support for this? Personally, I am OK with these functions raising an deprication error telling users to change the names of their functions. At least a very explicit and insistent user warning. @jkterry1 Thoughts?

Reply:
It breaks some stuff with supersuit, the idea is to have this here first, then update the one on supersuit, and then remove it altogether.

Reply:
In the past, when we have things that break functionality in supersuit, we simply release the two packages together. However, if you do want to go with this 3 step process, these functions should at least give a deprication warning.

Reply:
I will look into making a PR for supersuit in the near future then.